cl/users/email_handlers.py

```diff
@@ -416,8 +309,9 @@ def under_backoff_waiting_period(email_address: str) -> bool:
     :return bool: True if the email address is under a waiting period, if not
     False
     """
-    backoff_event = BackoffEvent.objects.filter(
+    backoff_event = EmailFlag.objects.filter(
         email_address=email_address,
+        flag_type=FLAG_TYPES.BACKOFF,
     ).first()
```

Comment:
If you're calling first(), you should be setting the ordering too.

Reply:
The same previous answer, since we have a DB constrain to allow only one Backoff event per email address, is it not necessary to order?